string user
string auladest
string msg
int64 pd
int64 idrob